Source/ARTDeviceDetails.h

```diff
@@ -7,12 +7,12 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface ARTDeviceDetails : NSObject
 
-@property (nonatomic) ARTDeviceId *id;
-@property (nullable, nonatomic) NSString *clientId;
-@property (nonatomic) NSString *platform;
-@property (nonatomic) NSString *formFactor;
-@property (nonatomic) NSMutableDictionary<NSString *, NSString *> *metadata;
-@property (nonatomic) ARTDevicePushDetails *push;
+@property (strong, nonatomic) ARTDeviceId *id;
+@property (strong, nullable, nonatomic) NSString *clientId;
+@property (strong, nonatomic) NSString *platform;
+@property (strong, nonatomic) NSString *formFactor;
+@property (strong, nonatomic) NSMutableDictionary<NSString *, NSString *> *metadata;
```

Comment:
I wonder why this was ever mutable. 🙄 ... I would have assumed this class should be immutable in aggregate.

Reply:
If set copy  for mutable object static analyzer shows a complain on it:
```
1. Property of mutable type 'NSMutableDictionary<NSString *,NSString *>' has 'copy' attribute; an immutable object will be stored instead
```